langchain4j-milvus/src/test/java/dev/langchain4j/store/embedding/milvus/MilvusEmbeddingStoreRemovalIT.java

```diff
@@ -25,6 +25,10 @@ class MilvusEmbeddingStoreRemovalIT extends EmbeddingStoreWithRemovalIT {
             .password(System.getenv("MILVUS_PASSWORD"))
             .consistencyLevel(STRONG)
             .dimension(384)
+            .idFieldName("id_field")
```

Comment:
Why is this change required? I can understand setting these fields explicitly in one of the tests, but this should not be required for all of them